Dockerfile

```diff
@@ -64,7 +64,7 @@ WORKDIR /data
 STOPSIGNAL SIGTERM
 
 # End user MUST set EULA and change RCON_PASSWORD
-ENV TYPE=VANILLA VERSION=LATEST EULA="" UID=1000 GID=1000
+ENV TYPE=VANILLA VERSION=LATEST EULA="" UID=1000 GID=1000 LC_ALL=en_US.UTF-8

```

Comment:
Have you tested this locale setting with the Ubuntu and Alpine based image builds?

Reply:
I have not been able to test this yet for the other distros. But since the tests pass as you said, it should be fine.